test/env/data.go

```diff
@@ -39,7 +39,7 @@ import (
 )
 
 const (
-	EventuallyTimeout      = "30s"
+	EventuallyTimeout      = "90s"
```

Comment:
question: Why are we bumping this? 30s seems like it should be plenty for this use case to me, but I'd happily bump if you've been running into problems

Reply:
I run this locally, if its the first time I bring up KIND cluster it will timeout. It took around 40s etc to finish all the testing in my local.